src/utils/s3_mods.py

```diff
@@ -0,0 +1,148 @@
+# General functions
+import json
+import pandas as pd
+from io import StringIO
+import logging
+from typing import List
+import subprocess
+import os
+
+# Functions specific to s3 bucket
+import boto3
+import raz_client
+from rdsa_utils.cdp.helpers.s3_utils import file_exists, create_folder_on_s3
+
+
+# set up logging
+s3_logger = logging.getLogger(__name__)
+
+
+def create_client(config):
+    client = boto3.client("s3")
+    raz_client.configure_ranger_raz(client, ssl_file=config["s3"]["ssl_file"])
+    return client
+
+
+# Read a CSV file into a Pandas dataframe
+def rd_read_csv(filepath: str, **kwargs) -> pd.DataFrame:
+    """Reads a csv from s3 bucket into a Pandas Dataframe using pydoop. 
+    If "thousands" argument is not specified, sets it to ",". 
```

Comment:
I know what you're saying here, but it doesn't make sense to a user. 'it' should be described.

Reply:
Expanded the doc string